src/LLVM_Runtime_Linker.cpp

```diff
@@ -690,12 +692,8 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
                 modules.push_back(get_initmod_gcd_thread_pool(c, bits_64, debug));
             } else if (t.os == Target::QuRT) {
                 modules.push_back(get_initmod_qurt_allocator(c, bits_64, debug));
-                modules.push_back(get_initmod_posix_error_handler(c, bits_64, debug));
```

Comment:
Why remove all these? I think they are needed?

Reply:
Whoops forgot to remove this after seeing the runtime itself below. I think I see why these are removed now.